app/gui/src/dashboard/components/dashboard/AssetRow.tsx

```diff
@@ -390,7 +384,7 @@ export function RealAssetInternalRow(props: RealAssetRowInternalProps) {
           // Assume the parent is the root directory.
           return true
         }
-        if (isTeamParentsPath(parent.parentsPath, userGroups?.map((team) => team.id) ?? [])) {
+        if (isTeamParentsPath(parent.parentsPath, [])) {
```

Comment:
removing all the `userGroups` stuff feels sketchy to me but i *suppose* it will be replaced by the `ensoPath` PR soon anyway

Reply:
Right, also usergroups are available in the `User` interface now
I don't really understand what does this code do though

Reply:
ohh, forgot about `User` interface. yeah the code needs to be refactored, the intention is to disallow drag and drop from team space back to user space

Reply:
Well, it's no longer a case

Reply:
See: https://github.com/enso-org/cloud-v2/issues/1819